walletlibrary/src/main/java/com/microsoft/walletlibrary/identifier/HolderIdentifier.kt

```diff
@@ -25,5 +25,5 @@ interface HolderIdentifier {
      * @param data The data to sign.
      * @return The signed data.
      */
-    fun sign(data: String): String
+    fun sign(data: ByteArray?): String
```

Comment:
Can this return a ByteArray to be in parity with iOS? and then an injected Identifier doesnt have to worry about converting the signature into the properly encoded string? 

Also, why is the input nullable?